setup.cfg

```diff
@@ -66,3 +66,13 @@ ignore =
     W391
 exclude = pyramid/tests/,pyramid/compat.py,pyramid/resource.py
 show-source = True
+
+[check-manifest]
+ignore =
+    .gitignore
+    PKG-INFO
+    *.egg-info
+    *.egg-info/*
+ignore-default-rules = true
+ignore-bad-ideas =
```

Comment:
Ah, okay that makes sense. Sorry, I searched for "ignore-bad-ideas" on Google and nothing really came up; hadn't realised to check the `check-manifest` docs.